index.js

```diff
@@ -42,6 +42,9 @@ module.exports = {
     // force no semicolons
     'semi': ['error', 'never'],
 
+    // force one or more spaces between template tag functions and their template literals
+    'template-tag-spacing': ['error', 'always'],
+
```

Comment:
sorry, this isn't the rule I'm looking for... to clarify, in Tram-One, the components look a lot like this
```javascript
html`
  <div>
    <h1>Hello World</h1>
    <SpecialFooter />
  </div>
`
```

What I would like to do is force the indentation in the template to be consistent with the rest of the project (in this case, one tab). In the example above, it would be 1 tab before the starter and ending div tag, and 2 tabs before the h1 and SpecialFooter.